plugins/mvn/README.md

```diff
@@ -0,0 +1,28 @@
+## The mvn Plugin
+
+The [mvn](https://github.com/robbyrussell/oh-my-zsh/tree/master/plugins/mvn) plugin provides many [aliases](#Aliases).
```

Comment:
There's no need to link to the plugin here, better link to the maven website and describe in a few words what it is and does.